src/typings/react-native.d.ts

```diff
@@ -310,6 +317,11 @@ declare module 'react-native' {
         overScrollMode?: string; //enum( 'always', 'always-if-content-scrolls', 'never' )
         // iOS
         scrollIndicatorInsets?: {top: number, left: number, bottom: number, right: number };
+        // Windows only
+        onKeyDown? : Function;
```

Comment:
Why do we need keydown/up methods in a scroll view? In the unlikely event that someone needs this, they can install the callbacks on a sub-view.

Reply:
RX.ScrollViewProps contains onKeyPress, (inherited from RX.ViewPropsShared, along with other props that don't really work in ScrollView...). onKeyPress is used at least in the VLV to implement app-side arrow based focus navigation.
The Web implementation passes onKeyPress to the backing <div> (just when using custom scrollbars, not sure why).
For Windows I decided to handle the onKeyDown bubbled React events from whatever "keyboard focusable" children of the scroll view.  

---
In reply to: [159786141](https://github.com/Microsoft/reactxp/pull/404#discussion_r159786141) [](ancestors = 159786141)